eng/pipelines/runtime.yml

```diff
@@ -819,8 +825,10 @@ jobs:
     platforms:
     - Windows_NT_x64
     - OSX_x64
+    - iOS_x64
     - Linux_arm64
     - Linux_x64
+    - Android_x64
```

Comment:
DOn't duplicate jobs. We need to be very careful with the number of jobs we add to our build matrix.